apps/antalmanac/src/components/buttons/MapLink.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useCallback } from 'react';
+import { Link } from 'react-router-dom';
+import { useMediaQuery } from '@mui/material';
+import { MOBILE_BREAKPOINT } from '../../globals';
+
+interface MapLinkProps {
+    buildingId: number;
+    room: string;
+    isDark: boolean;
+    setActiveTab: (tab: number) => void;
+}
+
+const MapLink: React.FC<MapLinkProps> = ({ buildingId, room, isDark, setActiveTab }) => {
```

Comment:
```suggestion
const MapLink = ({ buildingId, room, isDark, setActiveTab }: MapLinkProps) => {
```
chore: consistency in how we type components